src/main/log-exporter.ts

```diff
@@ -2,21 +2,25 @@ import fs from 'fs'
 import path from 'path'
 import archiver from 'archiver'
 import {status} from './status'
+import {MainStore} from './store'
 import {Config} from '../config/type'
-import {store} from './store'
 
-const getBackendLogPath = (config: Config): string =>
+const getBackendLogPath = (config: Config, store: MainStore): string =>
```

Comment:
Should it be documented, that this function is created each time log is going to be appended so there's implicit synchronization through store?